it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersSoleProprietorUtrControllerISpec.scala

```diff
@@ -5,64 +5,64 @@ import play.api.libs.ws.DefaultWSCookie
 import play.api.test.WsTestClient
 import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
 
-class BusinessPartnersCorporateBodyUtrControllerIntegrationSpec
+class BusinessPartnersSoleProprietorUtrControllerISpec
   extends Specifications with TestConfiguration {
 
-  "GET /form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference" should {
+  val route = "/business-partners/partner-self-assessment-unique-taxpayer-reference"
+
+  s"GET /business-partners/partner-self-assessment-unique-taxpayer-reference" should {
 
     "render the partnership-self-assessment-unique-taxpayer-reference page" in {
       given
         .commonPrecondition
 
       WsTestClient.withClient { client =>
-        val result = client.url(s"$baseUrl/form/business-partners/partnership-self-assessment-unique-taxpayer-reference")
+        val result = client.url(s"$baseUrl" + s"$route")
           .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
           .get()
 
         whenReady(result) { res =>
           res.status mustBe 200
           val page = Jsoup.parse(res.body)
-          page.title() must include("Does the partner have a Self Assessment Unique Taxpayer Reference (UTR)?")
-          page.getElementsByTag("h1").text() must include("Does test partner have a Self Assessment Unique Taxpayer Reference (UTR)?")
+          page.title() must include("What is the partner’s Corporation Tax Unique Taxpayer Reference (UTR)?")
+          page.getElementsByTag("h1").text() must include("What is {{partner name}}’s Self Assessment Unique Taxpayer Reference (UTR)?")
         }
       }
     }
   }
 
-  "POST /form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference" when {
+  "POST /business-partners/partner-self-assessment-unique-taxpayer-reference" when {
```

Comment:
Same as route comment above - could use the route here like you've done in the CorporateBodyISpec + missing prefix